src/foam/nanos/crunch/services.jrl

```diff
@@ -0,0 +1,120 @@
+p({
+  "class":"foam.nanos.boot.NSpec",
+  "name":"capabilityCategoryDAO",
+  "lazy":true,
+  "serve":true,
+  "serviceScript":"""
+    return new foam.dao.EasyDAO.Builder(x)
+      .setAuthorize(false)
```

Comment:
Shouldn't this be readOnly?

Reply:
should be, thanks, added ticket https://nanopay.atlassian.net/browse/NP-1334

Reply:
https://github.com/Nauna/foam2/pull/90